File: PRAK110-2310817210021-RAKHMAD FAUZHAN RAMADHAN.c
#include <stdio.h>
#include <math.h>

int main() {
    // Nilai alas dan tinggi
    double alas = 5;
    double tinggi = 12;

    // Menghitung panjang sisi A, sisi B, dan sisi C
    double sisi_a = tinggi;
    double sisi_b = sqrt(alas * alas + tinggi * tinggi);
    double sisi_c = alas;

    // Menghitung keliling
    double keliling = sisi_a + sisi_b + sisi_c;

    // Menghitung luas
    double luas = 0.5 * alas * tinggi;

    printf("Diketahui :\n");
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);
    printf("Jawab :\n");
    printf("Sisi A = %.0f cm\n", sisi_a);
    printf("Sisi B = %.0f cm\n", sisi_b);
    printf("Sisi C = %.0f cm\n", sisi_c);
    printf("Keliling = %.0f cm\n", keliling);
    printf("Luas = %.0f cm\n", luas);

    return 0;
}

File: MODUL 1/PRAK110-2310817210021-RAKHMAD FAUZHAN RAMADHAN.py
import math
alas = 5.0
tinggi = 12.0


sisi_a = tinggi
sisi_b = math.sqrt(alas * alas + tinggi * tinggi)
sisi_c = alas


keliling = sisi_a + sisi_b + sisi_c


luas = 0.5 * alas * tinggi


alas_str = str(alas).rstrip('0').rstrip('.')  
tinggi_str = str(tinggi).rstrip('0').rstrip('.')
sisi_a_str = str(sisi_a).rstrip('0').rstrip('.')
sisi_b_str = str(sisi_b).rstrip('0').rstrip('.')
sisi_c_str = str(sisi_c).rstrip('0').rstrip('.')
keliling_str = str(keliling).rstrip('0').rstrip('.')
luas_str = str(luas).rstrip('0').rstrip('.')

print("Diketahui :")
print(f"Alas = {alas_str} cm")
print(f"Tinggi = {tinggi_str} cm")
print("Jawab :")
print(f"Sisi A = {sisi_a_str} cm")
print(f"Sisi B = {sisi_b_str} cm")
print(f"Sisi C = {sisi_c_str} cm")
print(f"Keliling = {keliling_str} cm")
print(f"Luas = {luas_str} cm")
